File: adapters/maze-path/src/main/java/sk/slsp/workshop/mazesolver/adapters/mazepath/configuration/MazePathAdapterConfiguration.java
package sk.slsp.workshop.mazesolver.adapters.mazepath.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import sk.slsp.workshop.mazesolver.adapters.mazepath.adapter.MazePathInMemoryAdapter;

@Configuration
public class MazePathAdapterConfiguration {
	
	@Bean
	public MazePathInMemoryAdapter mazePathInMemoryAdapter() {
		return new MazePathInMemoryAdapter();
	}
	
}

File: adapters/maze-path/src/main/kotlin/sk/slsp/workshop/mazesolver/adapters/mazepath/configuration/MazePathAdapterConfiguration.kt
package sk.slsp.workshop.mazesolver.adapters.mazepath.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import sk.slsp.workshop.mazesolver.adapters.mazepath.adapter.MazePathInMemoryAdapter

@Configuration
class MazePathAdapterConfiguration {

	@Bean
	fun mazePathInMemoryAdapter() =
		MazePathInMemoryAdapter()

}
